src/App.tsx

```diff
@@ -16,48 +19,47 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const goods: string[] = [...goodsFromServer];
```

Comment:
copy of the array of goods should be inside the separate function and return the sorted goods instead of writing like so